chore: update links to monorepo web-components (#5017)
vaadin/flow-components: vaadin-spreadsheet-flow-parent/README.md

# Spreadsheet component for Vaadin Flow

> This project is the Component implementation of [`<vaadin-spreadsheet>`](https://vaadin.com/docs/latest/components/spreadsheet) for use from the server side with [Vaadin Flow](https://github.com/vaadin/flow).

Reviewer: ```suggestion
A spreadsheet component for [Vaadin Flow](https://github.com/vaadin/flow).

This is a port of: https://github.com/vaadin/spreadsheet
```
